src/core/Slider/Thumb.tsx

```diff
@@ -0,0 +1,105 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { Tooltip, TooltipProps } from '../Tooltip';
+import { CommonProps } from '../utils/props';
+
+/**
+ * Thumb is a local component used to show and modify the values maintained by the Slider.
+ * Only one Thumb can be active at a time. A Thumb is made active when the user selects
+ * it with pointer. Whenever a Thumb is active, focused, or hovered its tooltip is shown.
+ */
+export const Thumb = ({
+  value,
```

Comment:
Great job moving stuff to props!
but could you define just `ThumbProps` type here too? Would look nicer..